references/classification/latency_tensorflow.py

```diff
@@ -9,13 +9,13 @@
 import os
 import time
 
-import numpy as np
-import tensorflow as tf
-
 os.environ["USE_TF"] = "1"
 os.environ["TF_CPP_MIN_LOG_LEVEL"] = "2"
 
-from doctr.models import classification
+from doctr.models import classification  # isort: skip
```

Comment:
@odulcy-mindee We need to import from doctr before otherwise it defaults to keras v3